### PR TITLE
Fix bug where one connect failure causes all to fail

### PR DESCRIPTION
**Pull request by bdrosen96**

### CODE REVIEW

**Comment by bdrosen96** [comment]
@jettify can you review?

File: aioredis/sentinel.py [hunk 1]
```diff
@@ -175,10 +175,15 @@ def create_sentinel(sentinels, *, db=None, password=None,
...
-                                           password=password,
-                                           encoding=encoding, loop=loop)
-        sentinels_connections.append(sentinel)
+        try:
+            sentinel = yield from create_redis((hostname, port), db=db,
+                                               password=password,
+                                               encoding=encoding, loop=loop)
+            sentinels_connections.append(sentinel)
+        except OSError:
+            pass
```
Comment (jettify) [inline aioredis/sentinel.py]: I think we should add `warnings.warn` or add logging, it is good to know that one of sentinels is dead.

Comment (jettify) [inline aioredis/sentinel.py]: `OSError` has `errno`, probably useful information to log

Comment (bdrosen96) [inline aioredis/sentinel.py]: updated

**Comment by bdrosen96** [comment]
@jettify is this ok to merge?

**Comment by jettify** [comment]
lgtm

**Pull request merged**